README.md

```diff
@@ -6,17 +6,12 @@ for working with buckets, tables and workspaces are covered.
 
 ## Install
 
-`$ pip3 install git+https://github.com/keboola/sapi-python-client.git`
-
-or 
-
 ```bash
-$ git clone https://github.com/keboola/sapi-python-client.git && cd sapi-python-client
-$ python setup.py install
+pip install git+https://github.com/keboola/sapi-python-client.git
```

Comment:
just btw, it can now be installed with `pip install kbcstorage` https://pypi.org/project/kbcstorage/

Reply:
good point